jfr-streaming/build.gradle.kts

```diff
@@ -0,0 +1,62 @@
+val gsonVersion: String by project
+val slf4jVersion: String by project
+val otelVersion: String by project
+val otelAlphaVersion: String by project
+
+
+plugins {
+    id("com.github.johnrengelman.shadow") version "5.2.0"
+    id("java")
+}
+
+repositories {
+    mavenCentral()
+    mavenLocal()
+}
+
+java {
+    toolchain {
+        languageVersion.set(JavaLanguageVersion.of(17))
+    }
+}
+
+tasks.test {
+  useJUnitPlatform()
+  jvmArgs("-Djdk.attach.allowAttachSelf=true")
+}
+
+dependencies {
+    implementation(platform("io.opentelemetry:opentelemetry-bom:${otelVersion}"))
```

Comment:
The conventions automatically apply dependency management, it looks like you can remove all boms and versions from this file